### DISCUSSION TITLE
help wanted

### DISCUSSION THREAD

**Discussion by CokoJl1** [community/community#64545] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

The error occurred when executing the -S command in the file install_pkg.sh , line 67: command not found. This message appeared after the file was executed install.sh . Please help me. I have already tried to perform all the necessary actions manually, but everything works except the themes. I'm new to Hyprland and really want to get this configuration, it's so beautiful, but for some reason I can't install it the usual way. The only thing I use is Manjaro, but that probably has nothing to do with it.

**Comment by aurelienheude** [comment]
this is a test answer

**Comment by LinukaAr** [comment]
It seems like you're encountering an issue when running a shell script (`install_pkg.sh`) that contains a command not found error on line 67. This issue is likely causing problems during the installation process of a theme on your Manjaro system.

Here are some steps you can take to troubleshoot the issue:

1. **Check the Script:** Open the `install_pkg.sh` script and navigate to line 67. Look at the command present on that line and ensure that it is correctly typed and has no syntax errors.

2. **Command Not Found:** If you're encountering a "command not found" error, it generally means that the shell is unable to locate the command you're trying to execute. Double-check that the command name is correct and that the command is available in your system's PATH.

3. **Command Dependencies:** Some commands require certain dependencies or packages to be installed on your system before they can be used. Make sure that any required dependencies are already installed.

4. **Permissions:** Ensure that the script file (`install_pkg.sh`) has execute permissions. You can use the command `chmod +x install_pkg.sh` to make the script executable.

5. **Environment Variables:** If the script relies on certain environment variables, make sure they are correctly set before running the script.

6. **Debugging:** You can add `echo` statements to the script to print out variable values or messages. This can help you understand the flow of the script and identify where the issue is occurring.

7. **Shell Compatibility:** Check if the script is written for a specific shell (e.g., Bash, Zsh) and ensure that you're using the correct shell to execute the script.

8. **Check Line 67:** Look at the context around line 67 to see if there are any typos, missing arguments, or other issues that could be causing the error.

9. **Manual Installation:** If the issue persists, you might consider manually installing the themes or software mentioned in the script. This could involve running commands directly in the terminal instead of relying on the script.

10. **Script Updates:** If the script is part of a larger project or repository, check if there are any updates or discussions related to this issue on the project's GitHub repository or website.

11. **Reach Out for Support:** If you're still having trouble, consider reaching out to the maintainers of the theme or software. They might be able to provide specific guidance based on the script you're using and the Manjaro environment.

Remember that troubleshooting scripts can sometimes be complex, and it's important to have a backup of your system or data before making significant changes. If the issue is specific to the Hyprland theme, consider visiting their official documentation or support channels for more targeted assistance.
